arisa.json

```diff
@@ -441,6 +441,26 @@
           "Works As Intended"
         ]
       },
+      "removeSpam": {
+        "resolutions": [
+          "Awaiting Response",
+          "Cannot Reproduce",
+          "Done",
+          "Duplicate",
+          "Fixed",
+          "Incomplete",
+          "Invalid",
+          "Unresolved",
+          "Won't Fix",
+          "Works As Intended"
+        ],
+        "patterns": [
+          {
+            "pattern": "\\[~",
```

Comment:
Can we make this check that we are closing the ] in the same word too?

Also I think this won't trigger if it doesn't start with a ping

Reply:
> Can we make this check that we are closing the ] in the same word too?

Yeah, technically that would be possible too but it makes the regex more error prone and complicated, while not exactly making much of a difference. I don't think there's any reason for the string `[~` being used for anything other than a user mention.

> Also I think this won't trigger if it doesn't start with a ping

I don't see why that would be the case. I've tested it, just to make sure, and it appears to work fine. [MCL-16427](https://bugs.mojang.com/browse/MCL-16427)